### PR TITLE
new OrderService

### PR DESCRIPTION
**Pull request by Akmalov-Asror**

### CODE REVIEW

File: FurnitureShop/src/FurnitureShop.Client.Api/Services/OrderService.cs [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    public OrderService(UnitOfWork unitOfWork, AppDbContext context)
+    {
+        _unitOfWork = unitOfWork;
+        _context = context;
+    }
+
+    public async Task<OrderView> CreateOrder(CreateOrderDto createOrderDto)
+    {
+        if (createOrderDto is null)
+            throw new BadRequestException("createOrderDto is not null");
```
Comment (devdoston) [inline FurnitureShop/src/FurnitureShop.Client.Api/Services/OrderService.cs:27]: create order dto is null 
not kerak emas

Comment (Akmalov-Asror) [reply FurnitureShop/src/FurnitureShop.Client.Api/Services/OrderService.cs:27]: yoq kere

Comment (Akmalov-Asror) [reply FurnitureShop/src/FurnitureShop.Client.Api/Services/OrderService.cs:27]: teshirish kerekiuuuuuu???

Comment (devdoston) [reply FurnitureShop/src/FurnitureShop.Client.Api/Services/OrderService.cs:27]: OSHA gapdagi not sozi kerak emas

Comment (Akmalov-Asror) [reply FurnitureShop/src/FurnitureShop.Client.Api/Services/OrderService.cs:27]: haaa

**Pull request merged**